libs/iplant-clojure-commons/src/clojure_commons/exception.clj

```diff
@@ -26,102 +23,115 @@
    a ':errors' subkey in the response ':body'"
   [error-handler error-code]
   (fn [error error-type request]
-    (let [response (error-handler error error-type request)
+    (let [hndlr-resp (error-handler error error-type request)
           exception {:error_code error-code
-                     :reason (get-in response [:body :errors])}]
-      (header/content-type
-        (embed-error-info error
-                          exception
-                          (assoc response
-                            :body (cheshire/encode exception)))
-        "application/json; charset=utf-8"))))
+                     :reason (get-in hndlr-resp [:body :errors])}
+          response (header/content-type
+                     (assoc hndlr-resp :body (cheshire/encode exception))
+                     "application/json; charset=utf-8")]
```

Comment:
Also a minor thing and the previous code (also yours though, I think) did this too -- it might be easier to read this using the threading macro, especially if anything else gets added.

```clojure
(-> hndlr-resp
  (assoc :body (cheshire/encode exception))
  (header/content-type "application/json; charset=utf-8"))
```

(same applies a bit lower when you're doing `header/header` for `WWW-Authenticate`, and perhaps elsewhere)